hr_holidays_public/models/resource_mixin.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 Camptocamp - Telmo Santos
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from odoo import models
+
+
+class ResourceMixin(models.AbstractModel):
+    _inherit = "resource.mixin"
```

Comment:
```suggestion
    _inherit = "hr.employee"
```
what about adding this change not globally to mixin just to employees for instance? 
mixin is used in other objects and this behavior may not be wanted there

Reply:
Good idea. I agree